examples/10-crds.yaml

```diff
@@ -26,21 +27,105 @@ spec:
   conversion:
     strategy: None
   group: cert.gardener.cloud
+  subresources:
+    status: {}
   names:
     kind: Issuer
     listKind: IssuerList
     plural: issuers
+    singular: issuer
     shortNames:
       - issuer
-    singular: issuer
+  preserveUnknownFields: false
```

Comment:
This field is available only for `>= v1.15`. You could put an if around it.

Reply:
removed it, its more or less a luxury feature that we don't really need, please its set to false which should be the default.